src/System.Runtime/src/ApiCompatBaseline.uap101aot.txt

```diff
@@ -0,0 +1,291 @@
+Compat issues with assembly System.Runtime:
+TypesMustExist : Type 'System.AccessViolationException' does not exist in the implementation but it does exist in the contract.
```

Comment:
I think that argument can potentially be made for a lot of these types, right I know @SedarG is just trying to unblock some other work, but we should definitely make sure we have an issue tracking these so they don't get lost.